index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Marcelo Alarcon Portafolio</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/codeoutlinedprogrammingsigns_81143.ico"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <img src="images/Welcome.png" alt="logo" />
+      <img src="images/Union.png" alt="menu button" />
+    </header>
+
+    <section class="headline">
+      <div class="presentation_container">
+        <h2 class="primary_color">Hey there. I'm Marcelo</h2>
+        <h2 class="secondary_color">I'm a software developer</h2>
+      </div>
+
+      <div>
+        <p class="secondary_color">
+          I can help you build a product , feature or website Look
+          through some of my work and experience! <br />If you like
+          what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </p>
+      </div>
+
+      <div class="social_icons_container">
+        <img
+          class="social-icon"
+          src="images/twitter.svg"
+          alt="Twitter contact"
+        />
+        <img
+          class="social-icon"
+          src="images/linkedin.svg"
+          alt="Linkedin contact"
+        />
+        <img
+          class="social-icon"
+          src="images/medium.svg"
+          alt="Medium contact"
+        />
+        <img
+          class="social-icon"
+          src="images/github.svg"
+          alt="GitHub contact"
+        />
+        <img
+          class="social-icon"
+          src="images/angellist.svg"
+          alt="Angelist contact"
+        />
+      </div>
```

Comment:
- [ ]  For social media icons, Kindly I would recommend using ```ul``` tag to group them, and ```li``` per icon for a list of icons;
 each icon has an ```a``` tag. Apart from being semantically correct, the ```a``` tag will let you connect your social networks with the icons, using ```href```. Kindly Adjust it. :+1: :+1: